editor/editor_settings.cpp

```diff
@@ -374,6 +374,11 @@ void EditorSettings::_load_defaults(Ref<ConfigFile> p_extra_config) {
 	_initial_set("interface/editor/display_scale", 0);
 	// Display what the Auto display scale setting effectively corresponds to.
 	float scale = get_auto_display_scale();
+
+	_initial_set("interface/editor/enable_debugging_pseudolocalization", false);
+	set_restart_if_changed("interface/editor/enable_debugging_pseudolocalization", true);
+	// Use pseudolocalization in editor.
```

Comment:
Instead of this here, you can use `EDITOR_DEF_RST` where you actually use it in `TranslationServer` directly. The `_RST` part will also enable restart if changed. (See my other comment about caching the value too.)